.github/workflows/snyk-scan.yml

```diff
@@ -8,10 +8,14 @@ on:
       - 'v[0-9]+.[0-9]+.[0-9]+'
     branches:
       - main
+      - devops/snyk-fix
+
+permissions:
+  contents: read
 
 jobs:
   snyk_scan_test:
-    if: ${{ github.event_name == 'pull_request' }} || ${{ github.event_name == 'workflow_dispatch' }}
+    if: ${{ github.event_name == 'push' }} || ${{ github.event_name == 'workflow_dispatch' }}
```

Comment:
only on push?
Why do we have "on" pull_request above then?

Could you please give a quick summary of what this workflow shall achieve?

Reply:
That was only for testing purposes. It is fixed.